x/pricefeed/simulation/operations/msg.go

```diff
@@ -98,29 +123,29 @@ func getExpiryTime(ctx sdk.Context) (t time.Time) {
 	return t
 }
 
-// pickNewRandomPrice picks a new random price given the current price
```

Comment:
nit, I liked the comments here. Can you maybe readd them?

Reply:
@rhuairahrighairidh